build.sh

```diff
@@ -129,46 +128,51 @@ create_folders()
     info "Creating output folder structure..."
     mkdir -p "$FINAL_IMG_DIR"
     mkdir -p "$FS_MNT_POINT"
-    mkdir -p "$PARTS_DIR" "$PARTS_ARMBIAN_DIR" "$PARTS_SKYWIRE_DIR" "$PARTS_TOOLS_DIR"
+    mkdir -p "$PARTS_DIR" "$PARTS_ARMBIAN_DIR" "$PARTS_SKYWIRE_DIR"
     mkdir -p "$IMAGE_DIR"
 
     info "Done!"
 }
 
-get_tools()
-{
-  local _src="$ROOT/cmd/skyconf/skyconf.go"
-  local _out="$PARTS_TOOLS_DIR/skyconf"
-
-  info "Building skyconf..."
-  info "_src=$_src"
-  info "_out=$_out"
-  env GOOS=linux GOARCH=arm64 GOARM=7 go build -o "$_out" -v "$_src" || return 1
-
-  info "Done!"
-}
-
 get_skywire()
 {
-  local _DST=${PARTS_SKYWIRE_DIR}/skywire.tar.gz # Download destination file name.
-
-  if [ ! -f "${_DST}" ] ; then
-      notice "Downloading package from ${SKYWIRE_DOWNLOAD_URL} to ${_DST}..."
-      wget -c "${SKYWIRE_DOWNLOAD_URL}" -O "${_DST}" || return 1
-  else
-      info "Reusing package in ${_DST}"
-  fi
-
-  info "Extracting package..."
-  mkdir "${PARTS_SKYWIRE_DIR}/bin"
-  tar xvzf "${_DST}" -C "${PARTS_SKYWIRE_DIR}/bin" || return 1
-
-  info "Renaming 'hypervisor' to 'skywire-hypervisor'..."
-  mv "${PARTS_SKYWIRE_DIR}/bin/hypervisor" "${PARTS_SKYWIRE_DIR}/bin/skywire-hypervisor" || 0
-
-  info "Cleaning..."
-  rm -rf "${PARTS_SKYWIRE_DIR}/bin/README.md" "${PARTS_SKYWIRE_DIR}/bin/CHANGELOG.md"  || return 1
-
+#manually change the version when new packages are released - need automation for this!
+echo $(pwd)
+local _DST=${PARTS_SKYWIRE_DIR}/skywire-0.2.3-arm64.deb # Download destination file name.
+local _DST1=${PARTS_SKYWIRE_DIR}/skybian-skywire-0.1.2-arm64.deb
+local _DST2=${PARTS_SKYWIRE_DIR}/skybian-0.1.2-arm64.deb
+    if [ ! -f "/usr/bin/apt" ] ; then
+      if [ ! -f "${_DST}" ] ; then
+        notice "Downloading package from ${SKYWIRE_DOWNLOAD_URL} to ${_DST}..."
+        wget -c "${SKYWIRE_DOWNLOAD_URL}" -O "${_DST}" || return 1
+      fi
+      if [ ! -f "${_DST1}" ] ; then
+        notice "Creating skybian-skywire package from Makefile"
+        make skybian-skywire-package-arm64 || return 1
+        mv skybian-skywire-0.1.2-arm64.deb ${_DST1} || return 1
+      fi
+      if [ ! -f "${_DST2}" ] ; then
+        notice "Creating skybian package from Makefile"
+        make skybian-package-arm64 || return 1
+        mv skybian-0.1.2-arm64.deb ${_DST2} || return 1
+      fi
+    else
+      #apt exists and is assumed working
+      sudo apt install -y software-properties-common
+      #configure the package repository on the host
+      sudo add-apt-repository 'deb http://skyfleet.github.io/sky-update sid main'
+      curl -L http://skyfleet.github.io/sky-update/KEY.asc | sudo apt-key add -
```

Comment:
I'm sorry, we cannot add a community-maintained repository into our official build code.

Reply:
I didn't think so, but I don't have access to the official ones so this was done for testing and demonstration purposes until an official repository containing built packages is created

Reply:
Depends on https://github.com/SkycoinProject/skywire-mainnet/issues/387